js/src/components/inline-edit.component.js

```diff
@@ -0,0 +1,71 @@
+
+/**
+ * Vue component
+ * Allow user to edit a db record inline and send
+ * changes to server.
+ *
+ * Use an inline template.
+ *
+ * Properties need for this component are:
+ *
+ * context: string, a jQuery selector where the 'loading' class will be apply by semantic-ui;
+ *          - default to the requesting element.
+ * url:     string, the url to call;
+ * value:  array, array of value to send to server.
+ *
+ */
+
+export default {
+  name: 'atk-inline-edit',
+  props: {
+    url: String,
+    initialValue: String,
+    id: Number,
```

Comment:
i think ID should be a inside URL as a sticky get. The reasons are:
 - there may not be ID, we might want to edit something that has no ID, like "status" of a currently logged-in user.
 - ID is not something that component can change as it wishes.
 - If we introduce a special way to protect "get" arguments by signing them with limited-time token, this may not apply on "ID" and would require a separate protection.
 - similarly - form does not have "ID" as a separate get argument.

Reply:
On other hand - perhaps we can pass OTHER arguments, for example if i have multiple inline fields on a page and i want them all to communicate with the same callback JS. Or ability to reference value of other field when submitting. 

Perhaps the reason for ID field is so that we can use this component inside table, which would kinda justify the "ID" field. I'd prefer to have either "args" or nothing.

Reply:
Id value was not mandatory and thought it could be good to have. I removed it.